File: AD-HOC/1796.py
n = int(input())
resp = list(map(int, input().split()))
sat = 0
ins = 0

for i in range(n):
  if (resp[i] == 0):
    sat += 1
  else:
    ins += 1

if (sat > ins):
  print('Y')
else:
  print('N')

File: AD-HOC/1796.js
let n = Number(lines.shift());
let resp = lines.shift().split(' ').map(Number);
let sat = 0, ins = 0;

for (let i = 0; i < n; i++) {
  if (resp[i] == 0) {
    sat++;
  } else {
    ins++;
  }
}

if (sat > ins) {
  console.log('Y');
} else {
  console.log('N');
}
